service/create/service.go

```diff
@@ -694,6 +694,7 @@ func (s *Service) processCluster(cluster awstpr.CustomObject) error {
 			Clients:     clients,
 			HostClients: hostClients,
 		},
+		Logger: s.logger,
```

Comment:
The lack of a `New` function is a bit worrying, but I couldn't find any other struct creation locations.